client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -146,12 +140,7 @@ export default function SitesDashboard() {
 		// We need a category in the URL if we have a selected site
 		if ( sitesViewState.selectedSite && ! category ) {
 			setCategory( A4A_SITES_DASHBOARD_DEFAULT_CATEGORY );
-		} else if (
-			category &&
-			sitesViewState.selectedSite &&
-			selectedSiteFeature &&
-			selectedSiteFeature !== A4A_SITES_DASHBOARD_DEFAULT_FEATURE // If the selected feature is the default one, we can leave the url a little cleaner.
```

Comment:
For now, let's leave with the full route of the feature, we don't have a default. This should be provided by the feature family config (not implemented yet)